### DISCUSSION TITLE
粤梦缘 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#36] [模板讨论]
粤梦缘 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by chriskong5155** [comment]
模板提示出错了：
2023-08-04 07:37:27 
日志：Failed at 3/6 request, 
Fail assert: {"re": "200", "from": "status"} from success_asserts, 
Response Error : HTTP 302: Found, 
Request URL: https://www.dranime.net/

**Reply by wjf0214** [reply]
复现不出，请再次尝试看看。是否是网站原因导致的偶发事件？

**Reply by chriskong5155** [reply]
我重新拉了一遍模板再试也是一样的提示。我再试两天看看，先谢谢！！！

**Reply by wjf0214** [reply]
第三步和第一步基本上没有区别，你第一步过了，第三步出问题，可能第二步等有问题？账密是邮箱和密码没错吧？还不行一步一步单独执行看看返回。

**Reply by chriskong5155** [reply]
邮箱和密码没错的。我抓了HAR看了下，第二步post路径好像变了，尝试手动修改一下不成功 :（
新的post路径：
https://www.dranime.net/member.php?mod=logging&action=login&loginsubmit=yes&handlekey=login&loginhash=LKaXr&inajax=1

**Reply by wjf0214** [reply]
我看了下。之前登录可能是有点问题。虽说不确定我什么一直可以用。我调整了下，你更新下再试试。

**Reply by chriskong5155** [reply]
还是和之前一样的第三步报错，难道是我的718框架问题？

**Reply by wjf0214** [reply]
我也是最新的718，应该没关系。这判断不出是什么问题。你就点击模板一步一步测试。看看返回信息是否都正常。第一步的值获取是否成功，第二部是否登录成功。第三步看302指向哪里了。

**Reply by chriskong5155** [reply]
我把账号密码填进去一步一步测试到最后签到都行，但把1至3步连起来到第3步就提示出错，头都大了 ：（
第二步测试提示：
<script type="text/javascript" reload="1">if($(succeedmessage)) ($(succeedmessage).innerHTML = ";)if(typeofsucceedhandle =='function) succeedhandle (https://www.dranime.net/home.php?mod=space&do=notice&view=system,;</script> <scripttype="text/javascript">setTimeout("window.location.href ='https://www.dranime.net/home.php?mod=space&do=notice&view=system"3000):$(succeedmessage href).href =https://www.dranimenet/home.php?mod=space&do=notice&view=system'$(main message).style.display =none;$('main succeed.style.display = "5(succeedlocation)innerHTML = 欢迎您来，水之精灵LV3 ，现在将转入登录前页面;</script>
是否需要在第2步与第3步之间加个等待的参数呢？

**Reply by wjf0214** [reply]
你可以试试，加个3秒延迟看看。

**Comment by Tincichow** [comment]
大佬，失效了，跪求更新。

**Reply by wjf0214** [reply]
更新了。
